sirepo/package_data/static/js/raydata.js

```diff
@@ -637,25 +692,48 @@ SIREPO.app.directive('scanSelector', function() {
 		<div class="clearfix"></div>
                 <button type="submit" class="btn btn-primary" data-ng-show="showSearchButton()" data-ng-click="search()">Search</button>
 	      </form>
-              <table class="table table-striped table-hover col-sm-4">
-                <thead>
-                  <tr>
-                    <th data-ng-repeat="h in getHeader()">{{ h }}</th>
-                  </tr>
-                </thead>
-                <tbody ng-repeat="s in scans">
-                  <tr>
-                    <td><input type="checkbox" data-ng-checked="s.selected" data-ng-click="toggleScanSelection(s)"/></td>
-                    <td data-ng-repeat="c in getHeader().slice(1)">{{ getScanField(s, c) }}</td>
+              <div data-ng-if="searchStartTime && searchStopTime">
+	        <button class="btn btn-info btn-xs" data-ng-click="newCommand()" style="float: right;"><span class="glyphicon glyphicon-plus"></span></button>
```

Comment:
Rename newCommand to addColumn or something similar.

Reply:
e64969d8a3a9303ebcf97566971db0411515341f